src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];

```

Comment:
Calling 'Color.values()' multiple times can be inefficient as it creates a new array each time it is called. Consider storing the result of 'Color.values()' in a local variable and then using that variable to get the random color index.